Makefile

```diff
@@ -60,12 +60,6 @@ ORG ?= kuadrant
 # quay.io/kuadrant/kuadrant-operator-bundle:$VERSION and quay.io/kuadrant/kuadrant-operator-catalog:$VERSION.
 IMAGE_TAG_BASE ?= $(REGISTRY)/$(ORG)/kuadrant-operator
 
-ifeq (0.0.0,$(VERSION))
```

Comment:
Why removing this? this is pretty and neat way to set docker image tag when not `0.0.0`

Reply:
I was thinking the use case is covered here: https://github.com/Kuadrant/kuadrant-operator/pull/158/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L37-L41

Basically if the `VERSION` is 0.0.0 would enter the first conditional and set the tag to `latest`. If it's semantic, would set it to the `vx.y.z` and the default would be the default or the one passed.

Otherwise with the removed code it was setting the `IMAGE_TAG` to `v$(VERSION)` either when `VERSION` was a _sha_